src/module_1/module_1_meteo_api.py

```diff
@@ -166,43 +181,62 @@ def plot_data(df: pd.DataFrame, variable: str) -> None:
     # Retrieve the full name of the variable
     full_variable_name = short_to_full.get(variable, variable)
 
-    plt.figure(figsize=(12, 8))
-
-    # Create a distinct color for each city
-    colors = plt.cm.viridis(np.linspace(0, 1, len(COORDINATES)))
+    fig, ax = plt.subplots(figsize=(12, 8))
+    # Create a distinct color and line style for each city
+    colors = plt.cm.Set2(np.linspace(0, 1, len(COORDINATES)))
+    markers = ["o", "s", "D"]  # Circle, square, diamond
 
     for idx, city in enumerate(COORDINATES.keys()):
         city_data = df[df["city"] == city]
         mean = city_data[f"{variable}_mean"]
         std = city_data[f"{variable}_std"]
+        # Plot mean
         plt.plot(
             city_data["year"],
             mean,
-            label=city,
+            label=f"{city} Mean",
             color=colors[idx],
-            marker="o",
+            marker=markers[idx % len(markers)],
             linestyle="-",
             linewidth=2,
+            markersize=8,
         )
-        plt.fill_between(
-            city_data["year"], mean - std, mean + std, color=colors[idx], alpha=0.3
+        # Plot standard deviation with dashed lines
+        plt.plot(
```

Comment:
the point of fig, ax = plt.subplots() is to use "ax.plot()" here to gain control of whre to plot things in matplotlib. Plt will take the "current axes" but having the axes object ax we explicitly tells matplotlib where to plot.